iOS/Chameleon/component/component.yaml

```diff
@@ -13,4 +13,8 @@ libraries:
 samples:
   - name: "Chameleon Sample"
     path:  ../samples/ChameleonSample.sln
```

Comment:
Need to add the remove tag to remove the source project.  

https://github.com/xamarin/XamarinComponents/pull/311/files#diff-a928b56fc399e0e478f1ed32a6fd5ab6R22